skr/webhook/pkg/util/util.go

```diff
@@ -0,0 +1,62 @@
+package util
+
+import (
+	"flag"
+	"fmt"
+	"os"
+	"path"
+
+	ctrl "sigs.k8s.io/controller-runtime"
+
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/clientcmd"
+	clientcmdapi "k8s.io/client-go/tools/clientcmd/api"
+)
+
+func GetConfig(kubeConfig string, explicitPath string) (*rest.Config, error) {
```

Comment:
Can we replace this method with [this](https://github.com/kubernetes-sigs/controller-runtime/blob/7a5d60deb3b6a5d6cbec4ba9c56b60549ab9de67/alias.go#L78) and using like this:
```golang
import ctrl "sigs.k8s.io/controller-runtime"
restConfig := ctrl.GetConfigOrDie()
```